Source/WebCore/html/HTMLInputElement.cpp

```diff
@@ -1426,6 +1426,8 @@ ExceptionOr<void> HTMLInputElement::showPicker()
     if (!window || !window->hasTransientActivation())
         return Exception { ExceptionCode::NotAllowedError, "Input showPicker() requires a user gesture."_s };
 
+    window->consumeTransientActivation();
```

Comment:
```suggestion
    if (!window || !window->consumeTransientActivation())
        return Exception { ExceptionCode::NotAllowedError, "Input showPicker() requires a user gesture."_s };
```

This is the more common shorter pattern that is equivalent.